tests/rest/admin/test_user.py

```diff
@@ -3384,6 +3384,40 @@ def test_get_rooms_with_nonlocal_user(self) -> None:
         self.assertEqual(1, channel.json_body["total"])
         self.assertEqual([local_and_remote_room_id], channel.json_body["joined_rooms"])
 
+    def test_from_ts_param(self) -> None:
+        # Create rooms and join, grab timestamp of room creation
+        room_creation_timestamp = self.reactor.rightNow
+        other_user_tok = self.login("user", "pass")
+        number_rooms = 5
+        for _ in range(number_rooms):
+            self.helper.create_room_as(self.other_user, tok=other_user_tok)
+
+        # advance clock 48 hours
+        self.reactor.advance(60 * 60 * 48 * 1000)
+
+        # get a timestamp beginning 24 hours ago
+        current_time = self.reactor.rightNow
```

Comment:
I don't see `self.reactor.rightNow` used elsewhere

```suggestion
        current_time = self.hs.get_clock().time_msec()
```